tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -117,7 +116,7 @@ expression_matrix = transpose(
 )
```

Comment:
This can be written much simpler, e.g., something like
```
expression_matrix = randn(n_cells, n_genes)
expression_matrix[1:(ncells ÷ 2), 1:(n_genes ÷ 3)] .+= 10
expression_matrix[(ncells ÷ 2 + 1):end, (2*(n_genes ÷ 3) + 1):end] .+= 10
```
(the `transpose`/`vcat`/`hcat` stack is also quite fishy).

Reply:
yes, this is much more efficient and has been adopted here.